### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by FFroehlich**

### CODE REVIEW

File: petab/visualize/helper_functions.py [hunk 1]
```diff
@@ -469,10 +469,13 @@ def get_data_to_plot(vis_spec: pd.DataFrame,
 
         # check correct preqeuilibration condition
```
Comment (yannikschaelte) [inline petab/visualize/helper_functions.py]: could also fix this typo :)

**Comment by yannikschaelte** [comment]
don't know why tests did not run through. re-trying after merge-into

File: petab/visualize/helper_functions.py [hunk 2]
```diff
@@ -469,10 +469,13 @@ def get_data_to_plot(vis_spec: pd.DataFrame,
...
         pre_cond = m_data[PREEQUILIBRATION_CONDITION_ID][ind_meas[0]]
-        bool_preequ = (pre_cond == m_data[PREEQUILIBRATION_CONDITION_ID])
+        bool_preequ = (pre_cond ==
```
Comment (dweindl) [inline petab/visualize/helper_functions.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5049701)

File: petab/visualize/helper_functions.py [hunk 3]
```diff
@@ -469,10 +469,13 @@ def get_data_to_plot(vis_spec: pd.DataFrame,
...
         # check correct preqeuilibration condition
         pre_cond = m_data[PREEQUILIBRATION_CONDITION_ID][ind_meas[0]]
-        bool_preequ = (pre_cond == m_data[PREEQUILIBRATION_CONDITION_ID])
+        bool_preequ = (pre_cond == 
+                       m_data[PREEQUILIBRATION_CONDITION_ID].iloc[ind_meas])
         # special handling is needed, if preequilibration cond is left empty
         if isinstance(pre_cond, Number) and np.isnan(pre_cond):
-            bool_preequ = np.isnan(m_data[PREEQUILIBRATION_CONDITION_ID])
+            bool_preequ = np.isnan(
+              m_data[PREEQUILIBRATION_CONDITION_ID].iloc[ind_meas]
```
Comment (dweindl) [inline petab/visualize/helper_functions.py]: Codacy Issue found: [Wrong hanging indentation (add 2 spaces).](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5049701)

**Comment by dweindl** [comment]
> don't know why tests did not run through. re-trying after merge-into

That's on purpose, safety-wise. Running from forks may allow others retrieving your secrets or adding to your bill...

**Comment by yannikschaelte** [comment]
ah right, did not see it's forked. how to proceed in such cases? merge to internal non-develop and then on to develop?

**Comment by dweindl** [comment]
> ah right, did not see it's forked. how to proceed in such cases? merge to internal non-develop and then on to develop?

Yeah, i think that, or enabling actions on the fork.

File: petab/visualize/helper_functions.py (hunk 2 above)
Comment (dweindl) [inline petab/visualize/helper_functions.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5049701)

File: petab/visualize/helper_functions.py (hunk 3 above)
Comment (dweindl) [inline petab/visualize/helper_functions.py]: Codacy Issue found: [Wrong hanging indentation (add 2 spaces).](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5049701)

File: petab/visualize/helper_functions.py (hunk 2 above)
Comment (dweindl) [inline petab/visualize/helper_functions.py]:
```suggestion
        bool_preequ = (pre_cond ==
```

File: petab/visualize/helper_functions.py (hunk 3 above)
Comment (dweindl) [inline petab/visualize/helper_functions.py]:
```suggestion
                m_data[PREEQUILIBRATION_CONDITION_ID].iloc[ind_meas]
```

**Pull request merged**